src/views/LoginView.vue

```diff
@@ -35,15 +35,20 @@
         </p>
 
         <div class="icons mb-3">
-          <a :href="getOAuth2LoginUrl('google')" v-if="supportEmailAccounts">
-            <IconBase iconName="google" width="48" height="48" />
-          </a>
-          <a :href="getOAuth2LoginUrl('facebook')" :disabled="disableFacebook" v-if="supportEmailAccounts">
-            <IconBase iconName="facebook" width="48" height="48" />
-          </a>
-          <a :href="getOAuth2LoginUrl('microsoft')" :disabled="disableMicrosoft" v-if="supportEmailAccounts">
-            <IconBase iconName="microsoft" width="48" height="48" />
-          </a>
+
+          <!-- oauth2 login buttons -->
+          <span v-if="supportEmailAccounts">
+            <b-link
+              :disabled="provider.isDisabled"
+              :href="getOAuth2LoginUrl(provider.name)"
+              @click="preventLoginIfProviderDisabled($event, provider)"
+              v-for="(provider, index) in oAuth2Providers" :key="index"
```

Comment:
```suggestion
              v-for="(provider, index) in oAuth2Providers" 
              :key="index"
```

Can you move `:key` to its own line too so that it's consistent with the other props? Looks good to me otherwise, I'm not picky on whether we disable or hide them completely on ropsten.

Reply:
Done 👍